internal/pkg/kite/client.go

```diff
@@ -89,3 +110,31 @@ func (c *Client) GetVersion(ctx context.Context) (string, error) {
 
 	return verStr, nil
 }
+
+// SendPipelineFailure sends a pipeline failure event to KITE webhook
+func (c *Client) SendPipelineFailure(ctx context.Context, payload PipelineFailurePayload) error {
+	url := fmt.Sprintf("%s/api/v1/webhooks/pipeline-failure?namespace=%s", c.baseURL, payload.Namespace)
+
+	jsonPayload, _ := json.Marshal(payload)
+	fmt.Printf("Sending failure request to %s containing \n %s", url, string(jsonPayload))
```

Comment:
Wouldn't printf just print to stdout instead of structured logging? Would this be correctly forwarded to Splunk?